### ISSUE TITLE
remove host udev dependency 

### ISSUE THREAD

**Issue by Madhu-1** [gluster/gcs#141]
https://github.com/gluster/gluster-containers/pull/126

**Comment by Madhu-1** [comment]
@JohnStrunk @kotreshhr PTAL

**Comment by Madhu-1** [comment]
https://github.com/gluster/gcs/pull/133

**Comment by JohnStrunk** [comment]
Reading this issue, I'm confused.
@Madhu-1 Are you suggesting that the patch gluster/gluster-containers#126 that sets `obtain_device_list_from_udev = 0` in `lvm.conf` would fix the problem that we put a band-aid on via #133?

It looks like there is an lvm problem also accd to https://bugzilla.redhat.com/show_bug.cgi?id=1674485#c8 (linked from the BZ in the gluster-containers PR)

**Comment by Madhu-1** [comment]
@JohnStrunk yes you are right, LVM has an issue. i think downgrading lvm package fixes this  issue
